fl4health/clients/fenda_client.py

```diff
@@ -6,6 +6,8 @@
 
```

Comment:
I've put a ticket into our backlog for this, but after we get these calculations ready and merged, we should dedicate a bit of time to trying to simplify this implementation a bit, perhaps by breaking it into pieces etc. As it stands now, the FENDA code has become pretty complex and overloaded. Not necessary here, but in a follow up, I'll try to simplify a few things.

Reply:
I think for now I can just separate extra additions to base fenda under name of fenda_mkmmd_client. Then we can think about other refactoring options. 
Also same thing for moon_client.

Reply:
Sounds good. I don't think we need to do a lot for this PR. Just want to make a note that we can probably make some stuff "prettier" with some additional thought 🙂